### PR TITLE
Add GitHub actions workflow

### PR DESCRIPTION
**Pull request by mtgto**

### CODE REVIEW

**Comment by mtgto** [comment]
Already tested https://github.com/mtgto/tokite/pull/1

File: spec/dummy/config/database.yml [hunk 1]
```diff
@@ -13,6 +13,10 @@ development:
 
 test:
   <<: *default
+<% if ENV["GITHUB_ACTIONS"] %>
```
Comment (mtgto) [inline spec/dummy/config/database.yml:16]: GitHub actions add some environment variables automatically.
https://docs.github.com/ja/actions/learn-github-actions/variables#%E6%97%A2%E5%AE%9A%E3%81%AE%E7%92%B0%E5%A2%83%E5%A4%89%E6%95%B0

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    - uses: actions/checkout@v3
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: .ruby-version
+        bundler-cache: true
+    - name: Setup DB
+      run: |
+        RAILS_ENV=test bundle exec rails app:tokite:ridgepole:install app:tokite:ridgepole:apply
+    - name: Run tests
+      run: bundle exec rspec
```
Comment (nekketsuuu) [inline .github/workflows/test.yml:35]: nit: On CircleCI, RSpec is configured to show some slow tests in logs. How about doing the same in GitHub Actions?

https://github.com/cookpad/tokite/blob/39a543f8fcae9f017e11aa801efdb5439d5e2fc8/.circleci/config.yml#L29

**Review by nekketsuuu — APPROVED** [review]

File: .github/workflows/test.yml (hunk 2 above)
Comment (mtgto) [reply .github/workflows/test.yml]: Agree. https://github.com/cookpad/tokite/pull/54/commits/d6f71c2c9f3506b3de34146d2ed20406f3987a7a

**Pull request merged**